modules/mcuboot/CMakeLists.txt

```diff
@@ -152,7 +152,7 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
       )
   endif()
 
-  if (CONFIG_NRF53_UPGRADE_NETWORK_CORE)
+  if (CONFIG_NRF53_UPGRADE_NETWORK_CORE AND CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY)
```

Comment:
this does not hold... why ramdisk needed for netcore image transfer depends on placement of application core secondary slot? Unless there is good reason for it I suggest we fix it right after the review.

Reply:
@hakonfam, do you have any input on this one ?

Reply:
This cannot stay like it as it force external flash to be enabled which in general case has nothing to do with DFU. We use external flash ONLY because there is not enough room on internal flash. But this will not be valid for small applications or (possibly) future SoCs.

Let's track and fix it after the release. There is no point of continuing the discussion now. Create a ticket in Jira.

Reply:
We only need ramdisk if secondary slot is stored in external flash. This check ensures that you only enable ramdisk if you are on a nrf53 and you support updating the network core and the secondary slot (for both net core and app core) is stored in external flash. Which is correct IMO.

Remember that the only reason we need the ramdisk is because the network core cannot access external flash.